Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Domain/Action/Beckn/Cancel.hs

```diff
@@ -119,6 +119,9 @@ cancel req merchant booking = do
             reasonCode = Nothing,
             driverId = Nothing,
             additionalInfo = Nothing,
+            driverLat = Nothing,
```

Comment:
why do we have `Nothing` here? Can we also use current location?

Reply:
Its on driver side when user cancels we need to add to the customer side thats why left Nothing.